.github/workflows/ci.yml

```diff
@@ -14,6 +14,9 @@ jobs:
             run: |
               python -m pip install --upgrade pip
               if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+          - name: setup .env file
+            run: |
+              mv .env-template .env
```

Comment:
received errors when the tests execute. I had moved some of the constants in `constants.py` to `.env` so that values could be changed by user and then I use python decouple to read in the constants from `.env`. When I moved them to `.env` I didn't provide default values/they were undefined so tests/CI process failed. Looks like python decouple can provide default values so I could define default values using python decouple, get rid of default values in `.env` and get rid of this step in the CI process.

Reply:
ok nice, just checked and in one of our projects I listed them separately: https://github.com/PyBites-Open-Source/pybitesbooks/blob/develop/.github/workflows/python-app.yml - but yeah if you can get away with not setting some of those because you have sensible defaults that's nice